### PR TITLE
Add view on GitHub links for easier content editing from the docs

### PR DESCRIPTION
**Pull request by mdo**
Fixes #31293 (which should've been an issue).

Thoughts on backporting to v4?

Preview: https://deploy-preview-31339--twbs-bootstrap.netlify.app/docs/5.0/getting-started/introduction/

### CODE REVIEW

File: site/layouts/_default/docs.html [hunk 1]
```diff
@@ -15,7 +15,10 @@
       </div>
 
       <div class="bd-intro pt-md-3 pl-lg-4">
-        <h1 class="bd-title" id="content">{{ .Title | markdownify }}</h1>
+        <div class="d-md-flex flex-md-row-reverse align-items-center justify-content-between">
+          <a class="btn btn-link btn-sm mb-2 text-muted text-decoration-none bg-light" href="https://github.com/twbs/bootstrap/tree/main/site/content/{{ .Page.File.Path }}" title="View and edit this file on GitHub" target="_blank" rel="noopener">View on GitHub</a>
```
Comment (XhmikosR) [inline site/layouts/_default/docs.html:19]: We should use the repo config variable here

**Comment by XhmikosR** [comment]
I don't mind backporting it BTW.

Not sure about the style though. Maybe we should use `btn btn-outline-dark btn-sm mb-2` instead of `btn-link`?

**Comment by mdo** [comment]
Tweaked the styles from a custom-ish button to a `.btn-bd-light` button a la versions dropdown.

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
I need to revisit this in case we could use another variable instead of hardcoding the path.

**Pull request merged**